<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndMain</class>
 <widget class="QMainWindow" name="wndMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>501</width>
    <height>318</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>501</width>
    <height>360</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>uniVid</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/img/images/uniVid_wicon.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#btnUnir, #btnLimpiar, #btnSubir, #btnBajar, #btnAgregar, #btnQuitar{
	border: 0px;
}

#btnUnir:pressed, #btnLimpiar:pressed, #btnSubir:pressed, #btnBajar:pressed, #btnAgregar:pressed, #btnQuitar:pressed  {
   border: 2px solid rgba(0,0,0,0);
}

#btnUnir{
    background-color: #212D5A;
    border: 1px solid #212D5A;
    border-radius: 8px;
    color: white;
}

#btnUnir:hover{
	border: 1px solid #35496b;
	background-color: #35496b;
}
#btnUnir:pressed{
	border: 2px solid #1c273a;
	background-color: #212D5A;
}
#btnUnir:disabled,#btnLimpiar:disabled{
    border: 1px solid #888888;
    background-color: #888888;
    color: #4b4b4b;
}


#btnLimpiar{
    background-color: #4a685e;
    border: 1px solid #4a685e;
    border-radius: 8px;
    color: white;
}

#btnLimpiar:hover{
	border: 1px solid #6a9687;
	background-color: #6a9687;
}
#btnLimpiar:pressed{
	border: 2px solid #334740;
	background-color: #4a685e;
}

#lstArchivos{
   background-color: #becfe5; 
    color: #1c1f21;
}

#lstArchivos::item:selected {
 background-color: #486e87;
}

QProgressBar{
                border: 1px solid #223168;
                border-radius: 2px;
                text-align: center;
                color: #171d22;
                text-shadow: 0px 0px 10px black;
            }

            QProgressBar::chunk {
                background-color: #223168;
                width: 10px;
                margin: 0px;}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>64</width>
    <height>64</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>501</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>501</width>
     <height>360</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Unir Videos</string>
     </attribute>
     <widget class="QPushButton" name="btnBajar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/images/icon_dw.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QListWidget" name="lstArchivos">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>431</width>
        <height>181</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>431</width>
        <height>181</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>431</width>
        <height>181</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLimpiar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAgregar">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>0</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/images/add_icon108x112.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QProgressBar" name="loader">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUnir">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>210</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Unir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSubir">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/images/icon_up.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnQuitar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>170</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/img/images/del_icon108x112.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Reducir tama√±o</string>
     </attribute>
     <widget class="QPushButton" name="btnReducir">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>150</y>
        <width>471</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Reducir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnArchivoReducirSel">
      <property name="geometry">
       <rect>
        <x>362</x>
        <y>40</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Seleccionar</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtArchivoReducir">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblTamanio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Droid Sans</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0 MB</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="loader_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblTamanio2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Droid Sans</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0 MB</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionAgregar">
   <property name="text">
    <string>Agregar nuevo</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionQuitar_todos">
   <property name="text">
    <string>Quitar todos</string>
   </property>
  </action>
  <action name="actionArchivos_nanse">
   <property name="text">
    <string>Archivos.. √∫nanse!</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
